magento-research/pwa-studio
# feature: Possibility to close checkout form

## Description
This PR introduces the possibility to close checkout and continue shopping. It seems like the approach was designed previously but the reset button was not added to the form. This PR adds the "Continue Shopping" button which allows closing both drawer and checkout pane (there was no possibility to close checkout pane previously)

## Related Issue
Closes #848 

 ## Motivation and Context
This change is required since there's no possibility to close the checkout pane and put the cart back to the previous state (without checkout form opened)

## How Has This Been Tested?
- Add a product to the shopping cart
- Open the shopping cart and click on the checkout button
- You should see "Continue Shopping" button.
- By clicking on "Continue Shopping" the drawer is closed
- If you open the shopping cart once again, the checkout information pane is closed

## Screenshots (if appropriate):

![image](https://user-images.githubusercontent.com/2148959/51966910-2fe43c00-246e-11e9-90b9-d933f923f351.png)
